server/views/appeals/show.njk

```diff
@@ -0,0 +1,39 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% extends "../partials/layout.njk" %}
+
+{% set pageTitle = applicationName + " - " + pageHeading %}
+{% set mainClasses = "app-container govuk-body" %}
+
+{% block beforeContent %}
+    {{ govukBackLink({
+      text: "Back to timeline",
+      href: paths.applications.show({ id: application.id }) + "?tab=timeline"
+    }) }}
+{% endblock %}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
```

Comment:
We need a `<div class="govuk-grid-column-full">` here to go around the header. The div should be closed on line 26 before the `<div class="govuk-grid-column-three-quarters">`